server/catgenome/src/main/java/com/epam/catgenome/util/motif/MotifSearchIterator.java

```diff
@@ -45,6 +46,8 @@
     private static final byte LOWERCASE_G = 'g';
     private static final byte LOWERCASE_T = 't';
     private static final byte LOWERCASE_N = 'n';
+    private static final int MAX_SEARCH_VALUE = 1_000_000;
```

Comment:
we need a way to configure this value, lets move it to manager and make it @Value, moreover Integer,MAX_VALUE is useless because list already can have size no more than MAX_VALUE